xbmc/threads/test/TestEndTime.cpp

```diff
@@ -16,7 +16,7 @@ namespace
 void CommonTests(XbmcThreads::EndTime& endTime)
 {
   EXPECT_EQ(static_cast<unsigned int>(100), endTime.GetInitialTimeoutValue());
-  EXPECT_LE(static_cast<unsigned int>(0), endTime.GetStartTime());
+  EXPECT_LT(static_cast<unsigned int>(0), endTime.GetStartTime());
```

Comment:
Just for anyone wondering. This changes because the old method 0 was aloud if the static method was being initialised for the first time. With these changes the start time should never be 0.

Reply:
Would it be good to leave this comment as doc next to the line?

Reply:
I don't really think it's relevant. I would consider it a bug in the old code.